
Fixed removal of existing symlinks before creation.

  - Rene Rebe <rene@exactcode.de>

--- ./dosfsck/Makefile.vanilla	2005-01-22 00:52:02.398320408 +0100
+++ ./dosfsck/Makefile	2005-01-22 00:52:13.061699328 +0100
@@ -18,6 +18,7 @@
 	ln -s dosfsck $(SBINDIR)/fsck.msdos
 	ln -s dosfsck $(SBINDIR)/fsck.vfat
 	rm -f $(MANDIR)/fsck.msdos.8
+	rm -f $(MANDIR)/fsck.vfat.8
 	ln -s dosfsck.8 $(MANDIR)/fsck.msdos.8
 	ln -s dosfsck.8 $(MANDIR)/fsck.vfat.8
 
--- ./mkdosfs/Makefile.vanilla	2005-01-22 00:51:20.139744688 +0100
+++ ./mkdosfs/Makefile	2005-01-22 00:51:39.801755608 +0100
@@ -18,6 +18,7 @@
 	ln -s mkdosfs $(SBINDIR)/mkfs.msdos
 	ln -s mkdosfs $(SBINDIR)/mkfs.vfat
 	rm -f $(MANDIR)/mkfs.msdos.8
+	rm -f $(MANDIR)/mkfs.vfat.8
 	ln -s mkdosfs.8 $(MANDIR)/mkfs.msdos.8
 	ln -s mkdosfs.8 $(MANDIR)/mkfs.vfat.8
 
